docs/developer-docs/docs/build-an-agent/build-an-onchain-ai-agent/summary.md

```
@@ -0,0 +1,116 @@
+   - Order lifecycle
+   - Price monitoring
+   - Execution flow
+
+2. To implement automated Orders with price prediction, add these features:
+   - Prediction integration
+   - Time windows
+   - Complex conditions
+
+## Development complexity
+
+#### Automated Orders
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)